src/main/java/entities/GameState.java

```diff
@@ -0,0 +1,5 @@
+package entities;
+
+public class GameState extends PublishedMaze{
+
```

Comment:
add attributes;
- position: 2 element int array to represent x and y coordinate
- getPosition: function that returns position
- movePosition: function that will update position